src/econfig_server.erl

```diff
@@ -535,65 +534,36 @@ parse_ini_file(ConfName, IniFile) ->
             Msg = list_to_binary(io_lib:format(Fmt, [IniFilename])),
             throw({startup_error, Msg})
     end,
+    {match, Sections} = parse_ini_bin_to_sections(IniBin),
```

Comment:
the case where there are no sections in the file should be handled here. It's useful when you need to pass a local file that will contains the updates.